### ISSUE TITLE
TreeSelect: display: 'chip' doesn't have remove button (X) for selected nodes in PrimeReact

### ISSUE THREAD

**Issue by krotek20** [primefaces/primereact#4847]
### Describe the feature you would like to see added

I don't know if it's a bug or intentional, but there is no X button to remove certain selected node in display: 'chip'. This happened with Primereact.

### Additional context

https://dxt86y.csb.app/

**Comment by melloware** [comment]
I have a feeling it was intentional as you should be checking and unchecking items from the checklist not from both places?

**Comment by krotek20** [comment]
> I have a feeling it was intentional as you should be checking and unchecking items from the checklist not from both places?

Fair enough, but I think nowadays most of the users are used to see that X button to remove entries directly from the field.

**Comment by melloware** [comment]
I will leave it open for discussion by PrimeTek.  Also you might want to check what PrimeVue and PrimeNG do to see if its already implemented there or not?  That would also help determine of whether its a missing feature just in PR.

**Comment by melloware** [comment]
Closing this as it's not that way in PrimeVue and keeping the libraries consistent.
